image_width: 1024
image_height: 768
camera_name: miabot_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [1297.234575, 0, 477.676971, 0, 1293.852207, 394.634391, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.14928, 0.277583, -0.000632, -0.0067, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [1272.505371, 0, 472.268622, 0, 0, 1278.32666, 394.08135, 0, 0, 0, 1, 0]
